scripts/apps/vocabularies/constants.ts

```diff
@@ -0,0 +1,18 @@
+export const MEDIA_TYPES = {
+    media: {
+        id: 'media',
+        label: 'Media gallery'
+    },
+    related_content: {
+        id: 'related_content',
+        label: 'Related items'
+    },
+};
+
+export const MEDIA_TYPE_KEYS = Object.keys(MEDIA_TYPES);
```

Comment:
I think you should use IDs, since keys are simply labels which should be freely refactorable.

Reply:
Yeah, a lot of over-complications, ....
Let's keep as it is, it is not needed to have refactorable keys.

Reply:
Which naming do you think gives more information about the action `addItem(MEDIA_TYPES.media)` or `addItem(MEDIA_TYPES.gallery)`

`MEDIA_TYPES.media` is just bad naming. It doesn't say which type of media it is. You refused to rename it, because you don't want to write migration scripts which is fair. I then suggest a method without the need to write migration scripts, and you now say you don't need the ability to rename things? :sweat: